tests/integration/targets/ec2_lc/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+# defaults file for ec2_instance
+ec2_instance_name: '{{ resource_prefix }}-node'
+ec2_instance_owner: 'integration-run-{{ resource_prefix }}'
+ec2_instance_type: t2.micro
```

Comment:
Imo it's better to update this to a nitro instance. t3 or t3a. It's more future resist.

Reply:
Ok, no ec2 instance is finally deployed. So I guess it doesn't matter.